### PR TITLE
[TF] Fix dependency conflict

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

tensorflow-metadata 1.13.0

### Reason for changes

```
tensorflow-metadata 1.14.0 requires protobuf<4.21,>=3.20.3, but you have protobuf 3.19.6 which is incompatible.
```

### Related tickets

140478

### CODE REVIEW

**Comment by AlexanderDokuchaev** [comment]
weekly/job/eval_tf/382/

File: tests/tensorflow/requirements.txt [hunk 1]
```diff
@@ -12,4 +14,4 @@ virtualenv
 # filelock 3.12.3 requires typing-extensions>=4.7.1, but tensorflow 2.12.1 requires typing-extensions<4.6.0,>=3.6.6
 # TODO: remove after upgrade to TF 2.13
 filelock<3.12.3
-openvino-dev==2024.1
```
Comment (alexsu52) [inline tests/tensorflow/requirements.txt:15]: ?

Comment (AlexanderDokuchaev) [reply tests/tensorflow/requirements.txt]: version of openvino should be set only in constraints.txt

**Review by alexsu52 — APPROVED** [review]
LGTM

**Pull request merged**